src/one_dragon/base/config/user_config.py

```
@@ -0,0 +1,250 @@
+        """读取 YAML 文件并迁移到 SQLite。"""
+        try:
+            raw = file_path.read_text(encoding='utf-8')
+        except Exception:
+            log.warning("读取配置文件失败 %s", file_path, exc_info=True)
+            return None
+
+        data = {}
+        try:
+            import yaml
+            data = yaml.safe_load(raw) or {}
+        except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)